helm/hpcc/values.yaml

```diff
@@ -78,10 +78,17 @@ global:
         #service.beta.kubernetes.io/aws-load-balancer-internal: "true"
         service.beta.kubernetes.io/azure-load-balancer-internal: "true"
       type: LoadBalancer
+      # If ingress is specified, an ingress Network Policy will be created any pod implementing a service with this visibility
+      # Default allows ingress from anywhere, but more restrictive rules can be used if preferred.
+      # Ingress rules can also be overriden by individual services
+      ingress:
+      - {}
```

Comment:
I see this is the correct way from the docs, but wasn't obvious to me - this was ingress rule that meant allow all.
Perhaps add a comment next to it?

Reply:
There's a comment on line 82...

Reply:
it was the syntax itself meaning what line 82 says, that confused me.